code/drasil-utils/lib/Utils/Drasil/Document.hs

```diff
@@ -11,6 +11,14 @@ blank = text ""
 indent :: Doc -> Doc
 indent = nest 4
 
+-- | Helper which concatenates two Docs.
```

Comment:
Why not use `<>` directly then?

Reply:
<> could certainly be used directly 

`a +:+. b = a <> b <> text "."`

Sine I was creating a helper which concatenates two Docs and appends a period, I thought that a situation may arise where a helper for concatenating two Docs would also be needed. Similar to `foldlSent` in `Fold.hs`

Reply:
Updated the PR, removing unnecessary functions.

Reply:
The point is that `Doc` already has a rich API, which we should use directly. We perhaps under-use it. `Sentence` is our own type, so we're forced to provide combinators for it ourselves.